Sources/BraintreeVenmo/BTVenmoDriver_Internal.h

```diff
@@ -38,4 +38,9 @@
  */
 @property (nonatomic, assign) BOOL shouldVault;
 
+/**
+ Exposed for testing the payment method usage associated with this request
```

Comment:
I'm a little confused about why this need to be exposed for testing. Shouldn't tests already have access to the venmo request since it's passed into the `tokenize` method?

Reply:
You are right! We don't need it for testing.